cfme/tests/cloud/test_tenant.py

```diff
@@ -26,7 +26,7 @@ def tenant(provider, setup_provider, appliance):
             provider.mgmt.remove_tenant(tenant.name)
 
 
-@pytest.mark.meta(blockers=[BZ(1462137, forced_streams=["5.9", "5.8"])])
+@pytest.mark.meta(blockers=[BZ(1462137, forced_streams=["5.9"])])
```

Comment:
This can be completely removed since the bug is closed. Also, this test case is assigned to the machine, you can assign it to yourself if it belongs to your FA.

Reply:
@valaparthvi 
I agree with you but this bug is closed as ```WONTFIX```. Hence I had not removed this completely.
Although I am not FA owner for this test case.
As we are not testing against ```5.8```; So I removed only this part.

Reply:
@mnadeem92 Is this test case comes under your area: ```Multi-Tenancy```?
Can I assign this test to you?